tests/cpp_unit_tests/test_container.cpp

```diff
@@ -193,6 +193,20 @@ TEST_CASE("Test component container") {
         CHECK(const_container2.get_item_by_seq<C>(1).a == 66);
         CHECK(const_container2.get_item_by_seq<C>(2).a == 7);
     }
+
+    SUBCASE("Test get group index") {
+        CHECK(const_container.get_group_idx<C>() == 0);
+        CHECK(const_container.get_group_idx<C1>() == 1);
+        CHECK(const_container.get_group_idx<C2>() == 2);
+    }
+
+#ifndef NDEBUG
+    SUBCASE("Test get id by idx2d") {
+        CHECK(const_container.get_id_by_idx(Idx2D{0, 0}) == 1);
+        CHECK(const_container.get_id_by_idx(Idx2D{0, 1}) == 11);
+        CHECK(const_container.get_id_by_idx(Idx2D{0, 2}) == 111);
```

Comment:
does it make sense to also check `Idx2D{1, x}`?

Reply:
What about `Idx2D{2, x}`?

Reply:
well if you feel like it 🙃